spec/recipes/server_spec.rb

```diff
@@ -64,6 +64,11 @@
     )
   end
 
+  it 'accepts locale environment variables' do
+    expect(chef_run).to render_file('/etc/ssh/sshd_config').
+      with_content('AcceptEnv LANG LC_* LANGUAGE')
```

Comment:
If you implement it as attribute, could you please add the second test: to verify the attribute change? Good example is [here](https://github.com/dev-sec/chef-ssh-hardening/blob/74d0c6f97236464a31e303dcead8313f1b67438a/spec/recipes/server_spec.rb#L176-L193) for `allow_root_with_key` attribute